subsys/mgmt/mcumgr/transport/Kconfig.lorawan

```
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2024, Jamie McCrae
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# The Kconfig file is dedicated to the LoRaWAN transport of MCUmgr
+# subsystem and provides Kconfig options to control aspects of
+# the transport.
+#
+# Options defined in this file should be prefixed:
+#  MCUMGR_TRANSPORT_LORAWAN_
+
+menuconfig MCUMGR_TRANSPORT_LORAWAN
+	bool "LoRaWAN MCUmgr SMP transport"
+	depends on LORAWAN
+	help
+	  Enables handling of SMP commands received over LoRa.
+
+if MCUMGR_TRANSPORT_LORAWAN
+
+config MCUMGR_TRANSPORT_LORAWAN_FRAME_PORT
+	int "LoRaWAN SMP frame port"
+	range 0 120
+	default 2
+	help
+	  LoRaWAN download and uplink frame port used for communication. All messages received on
+	  this port will be treated as SMP packets.
+
+config MCUMGR_TRANSPORT_LORAWAN_CONFIRMED_PACKETS
```

Suggestion to clarify that we are talking about uplinks:
```suggestion
config MCUMGR_TRANSPORT_LORAWAN_CONFIRMED_UPLINKS
```

---

Fixed